packages/client/package.json

```diff
@@ -50,6 +50,7 @@
     "minimatch": "5.0.1",
     "ms": "2.1.2",
     "node-fetch": "2.6.1",
+    "recursive-readdir": "2.2.2",
```

Comment:
I don't know what shifted, but `client` seems to have an indirect dependency on `recursive-readdir`. When I run the cli tests locally, I'd get a failure saying this package (`client`) couldn't find `recursive-readdir`. I made sure I had yarn install and project built. The only thing that resolved it was to add the dependency here.

Any suggestions on how to dig deeper are most welcome.

Reply:
When trying to reproduce one last time to get the exact error message, it's building fine. Maybe I was in an inconsistent local state?